x-pack/plugins/rollup/public/crud_app/sections/job_list/detail_panel/detail_panel.js

```diff
@@ -96,7 +119,7 @@ export class DetailPanelUi extends Component {
       const isSelected = tab === panelType;
       renderedTabs.push(
         <EuiTab
-          onClick={() => openDetailPanel({ panelType: tab, jobId: id })}
+          onClick={() => this.handleTabClick(tab)}
```

Comment:
I think I prefer how you handled it below in the job table

```
onClick={() => {
  trackUserAction(tabToUserActionMap[tab]); // with "tabToUserActionMap" declared in the upper scope
  openDetailPanel({ panelType: tab, jobId: id }); // as before
}}
```

This removes the need of a click handler and is consistent with other places.

Reply:
I agree, done!